src/sentry/utils/sdk_crashes/sdk_crash_detector.py

```diff
@@ -35,7 +35,7 @@ def fields_containing_paths(self) -> Set[str]:
 
     def should_detect_sdk_crash(self, event_data: NodeData) -> bool:
         sdk_name = get_path(event_data, "sdk", "name")
-        if sdk_name and sdk_name != self.config.sdk_name:
+        if sdk_name and sdk_name not in self.config.sdk_names:
```

Comment:
Also applies to code before this PR, but shouldn't we skip detection when `sdk_name` is `None`?

```suggestion
        if sdk_name is None or sdk_name not in self.config.sdk_names:
```